guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -5,8 +5,26 @@ include::common/header.adoc[]
 
 = {UpgradingDocTitle}
 
+In this guide, the terms upgrade, update, and migrate have the following meanings:
+
+Upgrading::
+The process of advancing your {ProjectServer} and {SmartProxyServer} installations from a y-stream release to the next, for example {Project} {ProjectVersionPrevious} to {Project} {ProjectVersion}.
+For more information, see xref:upgrading_overview[].
+
+Updating::
+The process of advancing your {ProjectServer} and {SmartProxyServer} installations from a z-stream release to the next, for example {Project} {ProjectVersion}.0 to {Project} {ProjectVersion}.1.
+ifndef::foreman-deb,orcharhino[]
+For more information, see xref:updating_project[].
+endif::[]
+
+Migrating::
+The process of moving an existing {Project} installation to a new instance.
+ifdef::foreman-el,katello,satellite[]
+For more information, see xref:migrating-project-to-a-new-el-system_{context}[].
```

Comment:
It says it can't find `migrating-project-to-a-new-el-system_upgrade-guide`, yet it is exactly what the file should generate in https://github.com/theforeman/foreman-documentation/pull/1409/files#diff-734b146c05567ccc07fb30f4cf7522edd41e20b96bfd8a01256dc695aa78d506R1? I'm lost.

Reply:
Yep, so am I... :D